examples/2016-10-31/function_request_parameters/template.yaml

```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Simple API Endpoint configured using Swagger specified inline and backed by a Lambda function
+Resources:
+
+  MyLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs8.10
+      CodeUri: src/
+      Events:
+        GetApi:
+          Type: Api
+          Properties:
+            Path: /post
+            Method: POST
+            RequestParameters:
+              - method.request.header.Authorization:
+                  Required: True
+                  Caching: True
+              - method.request.query.type
```

Comment:
I did a test template with both and found that only the `query` parameter was added to the API successfully.  This is because the doc you referenced is for the Amazon extensions to OpenAPI/Swagger, but it turns out that the swagger docs say to use `query` instead of `querystring`.  The Amazon extension must make this conversion behind the scenes.  https://swagger.io/docs/specification/describing-parameters/#query-parameters

Reply:
On second thought, I decided for consistency sake to update this to use `querystring` even though that is not what the swagger docs say.  Users will more likely be familiar with the AWS usage that swagger usage.